activerecord/lib/active_record/connection_adapters/abstract_adapter.rb

```diff
@@ -454,6 +454,10 @@ def disable_extension(name)
       def enable_extension(name)
       end
 
+      # This is meant to be implemented by the adapters that support custom enum types
+      def create_enum(*)
+      end
```

Comment:
```suggestion
      def create_enum(*); end
```

Reply:
I wrote it this way to keep it consistent with all the methods around it.